packages/webcert/src/feature/certificate/Inputs/UeTextField.test.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import '@testing-library/jest-dom'
+import { render, screen } from '@testing-library/react'
+import * as redux from 'react-redux'
+import userEvent from '@testing-library/user-event'
+import { CertificateDataConfig, CertificateDataElement, CertificateDataValueType } from '@frontend/common'
+
+import UeTextField from './UeTextField'
+
+const useSelectorSpy = jest.spyOn(redux, 'useSelector')
+const useDispatchSpy = jest.spyOn(redux, 'useDispatch')
+
+useSelectorSpy.mockReturnValue({})
+useDispatchSpy.mockReturnValue(jest.fn())
+
+const mockQuestion = {
+  value: { type: CertificateDataValueType.TEXT },
+  config: ({ prop: '' } as unknown) as CertificateDataConfig,
+} as CertificateDataElement
```

Comment:
Could use the same `fakeTextFieldElement` function that is used in `UeTextField.stories.tsx`  to generate data.